<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>СППР на основі морфологічного аналізу</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="M1file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>baseAlternatives.csv</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="M2file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>mvzip.csv</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="M3file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>mvzip2.csv</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Введення файлів з даними&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Дані для морфологічної таблиці</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Дані для матриці математичних взаємозв’язків </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Дані для числової матриці узгодженості</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LaunchButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>ЗАПУСК РОБОТИ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Результати роботи&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Найімовірніший статус підприємства&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="NumberOfPeoducedFinal">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Виготовлені машини на місяць</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Локалізація</string>
    </property>
   </widget>
   <widget class="QLabel" name="LocalisationFinal">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>110</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Фінансовий стан</string>
    </property>
   </widget>
   <widget class="QLabel" name="FinancialStatusFinal">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="StableQualityFinal">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>140</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>140</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Стабільна якість</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Готовність йти назустріч</string>
    </property>
   </widget>
   <widget class="QLabel" name="ReliabilityFinal">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>180</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>180</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Надійність контрактів</string>
    </property>
   </widget>
   <widget class="QLabel" name="AnsweresFinal">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>300</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Найкращий варіант дій:</string>
    </property>
   </widget>
   <widget class="QLabel" name="BestDecisionFinal">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CalForMorphPossibilities">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>410</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Таблиця
ймовірностей
для характеристик</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CalForConfigurations">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>410</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Таблиця
ймовірностей
конфігурацій</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CalForDecisionsTable">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>410</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Таблиця
ефективності
рішень</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1044</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="Help">
    <property name="title">
     <string>Довідка</string>
    </property>
    <addaction name="separator"/>
    <addaction name="OpenHelp"/>
   </widget>
   <addaction name="Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_2">
   <property name="text">
    <string>Відкрити програму</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Довідка</string>
   </property>
  </action>
  <action name="OpenHelp">
   <property name="text">
    <string>Відкрити довідку</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
